decentraland/.prettierignore

```diff
@@ -0,0 +1 @@
+native-base-theme/
```

Comment:
What's the deal with this dir?

Reply:
Hmmm, since those files were auto-generated I'm not sure if we should change them, should we?

Reply:
It's a little weird they're included in the repo as opposed to being available in `node_modules`. I agree that we shouldn't change the auto-generated files unless we plan on essentially "forking" them and maintaining them outside of the dep - and I don't think we want to do that in the short-term or medium-term.

Reply:
I think this is the main question gating merging

Reply:
Without running the generation script, the module doesn't work. Maybe we can keep that way until we need to make any changes.

Reply:
What's the generation script? Is it mentioned here? https://docs.nativebase.io/docs/GetStarted.html

Reply:
Sorry @pcowgill , I've just made a double check about what I've said and the module **does** work without this folder. The script is this: `node node_modules/native-base/ejectTheme.js` (refs: https://docs.nativebase.io/Customize.html#theme-and-variables-headref) and is used to customized the default components. I've run that because I've faced a problem with fonts setup and I've thought that ejection was part of the solution.

Reply:
I've just committed the removal.

Reply:
Ah ok, I figured I'd check since it seemed a little odd. It makes sense that customization led you in that direction, but good to hear it wasn't strictly necessary!